spot_rl_experiments/spot_rl/envs/gaze_env.py

```diff
@@ -3,103 +3,194 @@
 # LICENSE file in the root directory of this source tree.
 
 
+import os
+import sys
 import time
+from typing import Dict, List
 
 import cv2
 import numpy as np
+import rospy
 from spot_rl.envs.base_env import SpotBaseEnv
 from spot_rl.real_policy import GazePolicy
-from spot_rl.utils.utils import construct_config, get_default_parser
+from spot_rl.utils.utils import (
+    construct_config,
+    get_default_parser,
+    map_user_input_to_boolean,
+)
 from spot_wrapper.spot import Spot, wrap_heading
 
 DEBUG = False
 
+DOCK_ID = int(os.environ.get("SPOT_DOCK_ID", 520))
+
+
+def parse_arguments(args=sys.argv[1:]):
+    parser = get_default_parser()
+    parser.add_argument(
+        "-t", "--target-object", help="input:string -> name of the target object"
```

Comment:
Please use type hint.

Reply:
Can you give an example for this? Sorry I dont understand it fully.

Reply:
parser.add_argument( "-t", "--target-object", type=str)

Reply:
Thank you Jimmy, addressed.